public/locales/mn/translation.json

```diff
@@ -0,0 +1,45 @@
+{
+  "website": {
+    "name": "Азиудын эсрэг үзэн ядалтаас үүдсэн гэмт хэргийн бүртгэл"
+  },
+  "location" : "Байршил",
+  "date_range" : "Огнооны муж",
+  "daily_count" : "Өдөр бүрийн тоо",
+  "monthly_count" : "Сар бүрийн тоо",
+  "load_more" : "Үргэлжлүүлэх",
+  "incident_count_by_state" : "Хэргийн тоо аймгаар",
+  "incident_table" : {
+    "total_incident" : "Нийт {{count}} тохиолдол",
+    "no_data" : "Мэдээлэл байхгүй",
+    "total_incident_plural" : "Нийт {{count}} тохиолдлууд",
+    "state" : "Аймаг",
+    "count" : "Тоо",
+    "count_10k_asian" : "Тоо/10K Азиуд",
+    "count_1mm" : "Тоо/1MM"
```

Comment:
1MM means 1 million. Is there a corresponding translation?

Reply:
Yeah, I understood that, K, M, Bs are commenly used in Mongolia in dashboards, of course we can translate million = сая, etc but it will become harder to understand I believe.